www/includes/easyparliament/page.php

```diff
@@ -19,59 +19,56 @@ class PAGE {
     public $within_stripe_sidebar = false;
 
     public function page_start() {
+        $this->checkForAdmin();
```

Comment:
The old code was wrapped in a if !page_started - are we happy that can't happen with the rewrite?